internal/pgmonitor/exporter_test.go

```diff
@@ -48,15 +50,45 @@ func TestGenerateDefaultExporterQueries(t *testing.T) {
 }
 
 func TestExporterStartCommand(t *testing.T) {
-	t.Run("OneFlag", func(t *testing.T) {
-		commandSlice := ExporterStartCommand([]string{"--testFlag"})
-		assert.DeepEqual(t, commandSlice[:3], []string{"bash", "-ceu", "--"})
-		assert.DeepEqual(t, commandSlice[4:], []string{"postgres_exporter_watcher", "--testFlag"})
-	})
+	for _, tt := range []struct {
+		Name              string
+		BuiltInCollectors bool
+		Flags             []string
+		Expect            func(t *testing.T, command []string, script string)
+	}{
+		{Name: "NoCollectorsNoFlags",
+			Expect: func(t *testing.T, _ []string, script string) {
+				assert.Assert(t, cmp.Contains(script, "--[no-]collector"))
+			},
+		},
+		{Name: "WithCollectorsNoFlags", BuiltInCollectors: true,
+			Expect: func(t *testing.T, _ []string, script string) {
+				assert.Assert(t, !strings.Contains(script, "collector"))
+			},
+		},
+		{Name: "MultipleFlags", Flags: []string{"--firstTestFlag", "--secondTestFlag"},
```

Comment:
flags and collector behavior are independent of each other, so this makes sense to me.